File: PSET1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Please enter height");
    }
    while (n <= 0);
    for (int i = 1; i <= n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            if (j < n - i)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: PSET6/mario.py
from cs50 import get_int  # type: ignore

n = get_int("Enter Height:")
while n < 1 or n > 8:
    n = get_int("Enter Height")
for i in range(1, n + 1):
    for j in range(n):
        if j < n - i:
            print(" ", end="")
        else:
            print("#", end="")
    print("\n", end="")
